docs/website/docs/developers/general/developer-overview.md

```diff
@@ -258,6 +258,17 @@ should be in the format `[shape]xtype=[value]`. For example:
 1x5x3x1xf32=15,14,13,12,11,10,9,8,7,6,5,4,3,2,1
 ```
 
+Reading from a numpy file:
+``` text
+--input=@input.npy
+```
+
+Reading from a binary file with passing the shape and dtype because raw binary
+files have no metadata:
+``` text
+--input=[shape]xdtype=@input.bin
+```
```

Comment:
This has some overlap with an earlier section: https://github.com/iree-org/iree/blob/a85b8ea0e48459bc4ec131fb61ba121c8695903c/docs/website/docs/developers/general/developer-overview.md?plain=1#L144-L152

Some repetition is fine, but maybe link to that part of the document for a concrete example? Or merge this and that section somehow. Could also mention that `iree-benchmark-module` and `iree-run-module` share many flags.